language-php.md

```diff
@@ -8,6 +8,69 @@ We decided to follow [PSR-2].
 [1]: http://www.phptherightway.com/
 [PSR-2]: https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md
 
+## SQL Queries
+MySQL keywords are always capitalized: SELECT, INSERT, UPDATE, WHERE, AS, JOIN, ON, IN, etc. Break up long queries into multiple lines for legibility, preferably breaking for each clause.
```

Comment:
? @johndavedecano should we indicate that this is only applicable when using `RAW Queries`? Since most of the frameworks that we use, have their own ORMs